Armadillo/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
             android:name=".playback.PlaybackService"
```

Comment:
Might not be a bad idea to add 

  <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
  <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />

as well in prep for android 14